styles.css

```diff
@@ -222,6 +222,333 @@ li {
   padding: 12px;
 }
 
+/* Work section */
+
+.Works {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 120px 0.001px;
+  background: #fdfdfd;
+  position: absolute;
+  width: 100%;
+  height: 3658.4px;
+  left: 0.001px;
+  top: 682px;
+  font-family: Crete Round, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 32px;
+  line-height: 52px;
+  color: #1c1a19;
+}
+
+hr.works-line {
+  display: flex;
+  color: black;
+  margin: auto;
+}
+
+.works-img img {
+  display: flex;
+  margin: auto;
+  width: 86vw;
+  padding-top: 30px;
+}
+
+.stories-headings {
+  margin-left: 28px;
+  font-family: Crete Round, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 35px;
+  line-height: 44px;
+}
+
+.stories-content {
+  width: auto;
+  height: auto;
+  padding-left: 25px;
+  padding-right: 15px;
+  top: 1px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 18px;
+  line-height: 24px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  margin: 12px 0.01px;
+  margin: auto;
+}
+
+.stories-lang-list {
+  /* Auto Layout */
+
+  display: grid;
+  grid-column-gap: 20px;
+  grid-template-columns: auto auto auto auto;
+  padding: 20px;
+  margin-top: -15px;
+}
+
+.stories-list {
+  background-color: rgb(255, 255, 255);
+  grid-column-gap: 20px;
+  border: 1px solid rgba(0, 0, 0, 0.8);
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 20px;
+  line-height: 20px;
+}
+
+.stories-list-1 {
+  background: #fff;
+  border: 1px solid #8993a4;
+  box-sizing: border-box;
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+  margin: 0.0001px 12px;
+}
+
+.stories-list-2 {
+  background: #fff;
+  border: 1px solid #8993a4;
+  box-sizing: border-box;
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+  margin: 0.0001px 12px;
+}
+
+.stories-list-3 {
+  background: #fff;
+  border: 1px solid #8993a4;
+  box-sizing: border-box;
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+  margin: 0.0001px 12px;
+}
+
+.stories-list-4 {
+  background: #fff;
+  border: 1px solid #8993a4;
+  box-sizing: border-box;
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+  margin: 0.0001px 12px;
+}
```

Comment:
- [ ] Let's fix the tech stack of the portfolio. First, let's group all these selectors since they're the same. You can do just `.stories-list-item` and then use that class to name all the elements of your list.

- [ ] Your design is currently overflowing out of the page and it doesn't match the provided design. Let's try fixing this by scaling the font size and you can also try removing the padding of the parent element.

![image](https://user-images.githubusercontent.com/79114233/138370677-4dc81eec-1b30-450a-af67-069c30fd3b5d.png)

With reduced font-size and not padding: 
![image](https://user-images.githubusercontent.com/79114233/138370880-bd63c321-4beb-411a-8e37-8e9e71eccb52.png)

My example is not a complete fix on making these elements, but it's a start. You can play around until you have a result that makes you happy. :slightly_smiling_face: